components/drag-overlay.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { MuxUploaderDrop } from '@mux/mux-uploader-react';
+
+type Props = {
+  dragActive?: boolean | undefined,
```

Comment:
issue(blocker): I'm not seeing `dragActive` get set anywhere in the codebase. If that's accurate, remove references and refactor accordingly.

Reply:
I think a relic from when I was troubleshooting something. Will remove if I find the same.

Reply:
Actually nope, it's [being used](https://github.com/muxinc/stream.new/pull/128/files#diff-aa98fd0757d0e1741503c50cfafb7726939d19819638bbe8e030a27adfec34a3R136). Will keep.